temporalcli/commandsmd/commands.md

```diff
@@ -386,12 +385,34 @@ temporal workflow start \
 
 ### temporal workflow terminate: Terminate Workflow Execution by ID or List Filter.
 
-TODO
+The `temporal workflow terminate` command is used to terminate a [Workflow Execution](/concepts/what-is-a-workflow-execution). 
+Canceling a running Workflow Execution records a `WorkflowExecutionTerminated` event as the closing Event in the workflow's Event History. 
+Workflow code is oblivious to termination. Use `temporal workflow cancel` if you need to perform cleanup in your workflow.
+
+Executions may be terminated by [ID](/concepts/what-is-a-workflow-id) with an optional reason:
+```
+temporal workflow terminate [--reason my-reason] --workflow-id MyWorkflowId
+```
+
+...or in bulk via a visibility query [list filter](/concepts/what-is-a-list-filter):
+```
+temporal workflow terminate --query=MyQuery
+```
+
+Use the options listed below to change the behavior of this command.
+
+#### Options
+
+* `--workflow-id`, `-w` (string) - Workflow Id. Either this or query must be set.
+* `--run-id`, `-r` (string) - Run Id. Cannot be set when query is set.
+* `--query`, `-q` (string) - Start a batch to terminate Workflow Executions with given List Filter. Either this or Workflow Id must be set.
+* `--reason` (string) - Reason for termination. Defaults to message with the current user's name.
+* `--yes`, `-y` (bool) - Confirm prompt to perform batch. Only allowed if query is present.
```

Comment:
```suggestion
Includes options set for [single workflow or batch](#options-set-for-single-workflow-or-batch).
```

Or are you making this a completely separate options set just because of the English description of reason?

Reply:
Note that `#options-set-for-single-workflow-or-batch` mentions signal and reason is applied only to batch.

Reply:
Here's the discussion Roey and I had about this @cretz: https://github.com/temporalio/cli/pull/436#discussion_r1480582673

Reply:
We can make docs more generic if we need to, but I have no problem copy/pasting just for docs-only reasons if we have to like here.